cfgov/v1/models/images.py

```diff
@@ -87,6 +87,14 @@ def default_alt_text(self):
         # Override Wagtail default of setting alt text to the image title.
         return self.alt
 
+    # If the image is both large and its height-to-width ratio is approximately
+    # 1/2 we instruct the template to render large Twitter cards
+    # See https://dev.twitter.com/cards/types/summary-large-image
+    @property
+    def summary_large_image(self):
```

Comment:
I guess the only reason I don't like that as much is that, in the definition it's clear the image needs to be both large and have a 2:1 aspect ratio.  If someone were quickly looking at the code, they might be like, "ok cool we display the large image for large images" and then get confused why that's not working.  I don't want to be a blocker on this but I think more verbose is better.